test/boost/schema_change_test.cc

```diff
@@ -779,6 +779,7 @@ future<> test_schema_digest_does_not_change_with_disabled_features(sstring data_
     }
     cql_test_config cfg_in(db_cfg_ptr);
     cfg_in.disabled_features = std::move(disabled_features);
+    cfg_in.run_with_raft_recovery = true;
```

Comment:
Would be good to explain in comment why we need this -- the explanation in commit message is good but will be hard to find